tests/acceptance/language-select-test.js

```diff
@@ -17,7 +17,7 @@ test('setting a language preference persists after logout', (assert) => {
     });
 
     andThen(() => {
-      assert.equal(find('.view-current-title').text(), english.dashboard.title, 'Title is NOT in English after first log in');
+      assert.equal(find('.view-current-title').text(), english.dashboard.title, 'Title is in English after first log in');
```

Comment:
That's my old test I wrote, but I didn't realise that a description should contain 'what I expect' and not 'what went wrong'